node/resolvers/orderForm.ts

```diff
@@ -259,4 +259,24 @@ export const mutations = {
 
     return orderFormWithPayments
   },
+
+  itemsOrdination: async (
+    _: unknown,
+    { ascending, criteria }: ItemsOrdinationArgs,
+    ctx: Context
+    ): Promise<CheckoutOrderForm> => {
+
+    const {
+      clients: { checkout },
+      vtex: { orderFormId },
+    } = ctx
```

Comment:
You should use the orderFormId from the mutation input, with a default of the id from cookies

```suggestion
    const { clients: { checkout }, vtex } = ctx
    const {
      ascending,
      criteria,
      orderFormId = vtex.orderFormId,
    } = args
```